tlz/fips.go

```diff
@@ -1,10 +1,13 @@
-//go:build goexperiment.opensslcrypto
+//go:build goexperiment.opensslcrypto && requirefips
 
 package tlz
 
-import "crypto/boring"
+import (
+    "crypto/boring"
```

Comment:
The openssl bindings patch crypto/boring to call OpenSSL functions like this https://github.com/microsoft/go/blob/v1.24.4-1/patches/0002-Vendor-crypto-backends.patch